$Id$

Flush standard output after processing each event, since xev's exit(0) is
unreachable code.

This patch by Branden Robinson.  Thanks to Bill Currie for identifying the
problem and the solution.  See Debian bug #238638.

Index: xbase-clients/xev-X11R7.0-1.0.1/xev.c
===================================================================
--- xbase-clients.orig/xev-X11R7.0-1.0.1/xev.c	2006-02-18 14:56:32.000000000 -0500
+++ xbase-clients/xev-X11R7.0-1.0.1/xev.c	2006-02-26 19:50:37.000000000 -0500
@@ -1045,6 +1045,7 @@
 	    printf ("Unknown event type %d\n", event.type);
 	    break;
 	}
+    fflush(stdout);
     }
 
     XCloseDisplay (dpy);
